src/main/java/pl/wrapper/parking/exception/GlobalExceptionHandlerImpl.java

```diff
@@ -5,6 +5,7 @@
 import org.springframework.web.bind.annotation.ControllerAdvice;
 import org.springframework.web.bind.annotation.ExceptionHandler;
 
+
 @ControllerAdvice
 public class GlobalExceptionHandlerImpl implements GlobalExceptionHandler{
```

Comment:
- add logging on error with @Slf4j (log.error("some error message")) when an exception is thrown
- add handling for exceptions from responseHandler (like PwrApiNotRespondingException and ClassCastException)

Reply:
Also, do we really need handling for exceptions such as nullptr and IndexOutOfBounds? Neither of could be thrown in our application if code is written correctly. Maybe it would be better to remove the ones we don't need.